daemon/mgr/container_stats.go

```diff
@@ -32,48 +32,46 @@ func (mgr *ContainerManager) StreamStats(ctx context.Context, name string, confi
 	}
 
 	outStream := config.OutStream
-	if !c.IsRunningOrPaused() {
-		return errors.New("can only stats running or paused container")
-	}
 
 	var preCPUStats *types.CPUStats
 
-	wrapContainerStats := func(timestamp time.Time, metric *cgroups.Metrics) (*types.ContainerStats, error) {
-		stats := toContainerStats(timestamp, c, metric)
+	wrapContainerStats := func(metricMeta *containerdtypes.Metric, metric *cgroups.Metrics) (*types.ContainerStats, error) {
+		stats := toContainerStats(c, metricMeta, metric)
+
 		systemCPUUsage, err := getSystemCPUUsage()
 		if err != nil {
 			return nil, err
 		}
 		stats.PrecpuStats = preCPUStats
 		stats.CPUStats.SyetemCPUUsage = systemCPUUsage
 		preCPUStats = stats.CPUStats
+
 		networkStat, err := mgr.NetworkMgr.GetNetworkStats(c.NetworkSettings.SandboxID)
 		if err != nil {
 			// --net=none or disconnect from network, the sandbox will be nil
-			logrus.Warnf("sandbox not found, name = %s, err= %v", name, err)
+			logrus.Debugf("failed to get network stats from container %s: %v", name, err)
```

Comment:
I think hold the fail reason here could be better.

Reply:
Actually I cannot quite follow you. @ZYecho 
Could you be in details?

Reply:
I mean add the fail reason into the log, such as
`logrus.Debugf("sandbox not found, failed to get network stats from container %s: %v", name, err)`

Reply:
I got what you mean. @ZYecho   But "sandbox not found" may be not the fail reason. Besides, this reason would be  included in the `err`,
For example, https://github.com/alibaba/pouch/blob/master/daemon/mgr/network.go#L453 here, if sandbox is exactly not found, it returns `not found` https://github.com/alibaba/pouch/blob/master/vendor/github.com/docker/libnetwork/controller.go#L1198